src/appleseed/renderer/modeling/bsdf/hairbsdf.cpp

```diff
@@ -171,7 +171,7 @@ namespace
     {
         return std::accumulate(retAp.begin(), retAp.end(), 0.0f, [](float s, const Spectrum& ap)
         {
-            return s + luminance(ap.to_rgb(g_std_lighting_conditions));
+            return s + luminance(ap.illuminance_to_ciexyz(g_std_cmf));
```

Comment:
Regression, you're now computing luminance on a CIE XYZ triplet. Please double-check the rest of the PR for similar regressions (because I assume they may have been introduced with global search & replace operations).